### PR TITLE
install: allow libvirt ssh key authorization

### PR DESCRIPTION
**Pull request by snir911**
by passing the ssh private key to the container

Signed-off-by: Snir Sheriber <ssheribe@redhat.com>
Fixes: #186 

**this is WIP as i didn't test it doesn't break the other setup yet**

### CODE REVIEW

File: install/yamls/ccruntime-peer-pods-in-pod.yaml [hunk 1]
```diff
@@ -30,12 +30,20 @@ spec:
             name: peer-pods-cm
         command: ["/usr/local/bin/entrypoint.sh"]
         volumeMounts:
+        - name: ssh
+          mountPath: "/root/.ssh/"
+          readOnly: true
```
Comment (bpradipt) [inline install/yamls/ccruntime-peer-pods-in-pod.yaml:35]: @snir911 is it possible to have the ssh hostpath volume mount as a kustomize overlay only for libvirt ?

Comment (snir911) [reply install/yamls/ccruntime-peer-pods-in-pod.yaml:35]: right, done

**Comment by bpradipt** [comment]
@snir911 I think the WIP label can be removed since now the changes are localised to libvirt only by using libvirt specific kustomization.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**